index.html

```diff
@@ -0,0 +1,179 @@
+            <h2>Lecture</h2>
+            <p>
+              Listen to the speakers from various contries about the messages of
+              sharing and openning.
+            </p>
+          </div>
+          <div class="program-sche">
+            <img src="images/program2.png" />
+            <h2>Math Events Exhibition</h2>
+            <p>
+              Appreciate various creations applying CC license of artists,
+              organized from art center Nabi.
+            </p>
+          </div>
+          <div class="program-sche">
+            <img src="images/program3.png" />
+            <h2>Forum</h2>
+            <p>
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+          <div class="program-sche">
+            <img src="images/program4.png" />
+            <h2>Workshop</h2>
+            <p>
+              Try creating your own work using open source license rather than
+              just wathcing
+            </p>
+          </div>
+          <div class="program-sche">
+            <img src="images/program5.png" />
+            <h2>Math Events Ignite</h2>
+            <p>
+              Get opportunities to network with Math Events affiliates around
+              the world, also after the summit.
+            </p>
+          </div>
+        </div>
+        <a href="#" class="see">SEE THE WHOLE PROGRAM</a>
+        <a href="#" class="join">Join MATH EVENTS 2023</a>
+      </div>
+    </div>
+
+    <div class="speakers">
+      <div class="container3">
+        <h2>Featured Speakers</h2>
+        <hr class="line2" />
+        <br />
+        <div class="featured-speak">
+          <div class="featured-speakers"></div>
+          <button class="more">MORE ↓</button>
+          <div class="featured-speakers2"></div>
+        </div>
+      </div>
+
+      <div class="sponsors">
+        <div class="container4">
+          <h5>Partner</h5>
+          <hr class="line3" />
+          <div class="brands">
+            <a href="#" class="mozilla">mozilla</a>
+            <a href="#">Google</a>
+            <a href="#">NAVER</a>
+            <a href="#">daumkakao</a>
+            <a href="#">airbnb</a>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <footer class="footer-desk">
+      <div class="container5">
+        <div class="footer">
+          <img src="images/footer.png" />
+          <div class="footerr">
+            <p class="footer-p">
+              <b>2023 Creative Commons Korea. Some Rights Reserved.</b>
+            </p>
+            <br />
+            <p class="footer-p1">
+              <b
+                >달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈
+                저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.
+                자세한 이용기능 내용과 허락조건은
+                http://creativecommons.org/licences/by/2.0/kr 를 보세요.</b
+              >
+            </p>
+          </div>
+        </div>
+      </div>
+    </footer>
```

Comment:
You have done very well in fixing all the issues that the previous reviewer pointed out 👍. However, I noticed that the footer of the homepage does not show on the mobile screen. Perhaps it was a bug that was mistakenly introduced while you were fixing the issues the previous reviewer pointed out. Still, I think you should first look at your CSS file and ensure that all your media queries are intact and doing what is expected. I encourage you to contact me on Slack if you need help with this and I will be glad to help 💪. Remember, being consistent with the design template is a requirement 😉.

![image](https://github.com/meryemsanem/capstone-project/assets/107795045/9648fa49-aecd-49cb-8657-282e59ced0e2)

Reply:
Hey, I think on the design there is no footer for this part.. So I think this one is the right design....